.github/labeler.yml

```diff
@@ -1,2 +1,8 @@
 documentation: 
 - 'docs/**/*'
+
+dependencies:
```

Comment:
Can we change this to something else? They're not actually dependencies. Maybe `project` or `project_version`? Something like that

Reply:
Definitely. This was just picked from the current labels. There are also potential variations we care about (adding, removing, and updating).